internal/handler/handlers.go

```diff
@@ -1630,7 +1642,13 @@ func IndexPageHandler(svr server.Server, jobRepo *job.Repository) http.HandlerFu
 
 func PermanentRedirectHandler(svr server.Server, dst string) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		svr.Redirect(w, r, http.StatusMovedPermanently, fmt.Sprintf("%s%s/%s", svr.GetConfig().URLProtocol, svr.GetConfig().SiteHost, dst))
+		profile, _ := middleware.GetUserFromJWT(r, svr.SessionStore, svr.GetJWTSigningKey())
```

Comment:
any reason why you changed this function? changing the other handlers would be enough unless i am missing something?